docs/source/locking.rst

```diff
@@ -0,0 +1,80 @@
+Locking
+=======
+
+Locking an object
+-----------------
+
+To lock an object send a POST request to the ``/@lock`` endpoint.
```

Comment:
@buchi I'd add the information that the lock endpoint can be called on any object in Plone. Otherwise the reader might think this is a root-level endpoint.